src/tribler-core/tribler_core/components/tests/test_tribler_components.py

```diff
@@ -26,6 +26,7 @@
 
 # pylint: disable=protected-access
 
+@pytest.mark.asyncio
```

Comment:
Well, this and some other files already contain `pytestmark = pytest.mark.asyncio` which is equivalent to putting `pytest.mark.asyncio` to _every_ test in the file...

Reply:
Fixed.